tools/CompressUXMetadata.ps1

```diff
@@ -0,0 +1,69 @@
+# ----------------------------------------------------------------------------------
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+# Move the UX metadata files under modules to the artifacts folder and compress them into a zip file.
+
+$RootPath = "$PSScriptRoot/.."
+$RepoArtifacts = 'artifacts'
+$UXMetadataFolder = "$RootPath/$RepoArtifacts/UX"
+If (-Not (Test-Path -Path $UXMetadataFolder))
+{
+    New-Item -ItemType Directory -Path $UXMetadataFolder
+}
+Else
+{
+    Get-ChildItem $UXMetadataFolder | Remove-Item -Force -Recurse
+}
+
+$Modules = Get-ChildItem -Path "$RootPath/src" -Exclude "lib" -Directory | ForEach-Object { $_.BaseName }
```

Comment:
"shared" is also an exception

Reply:
Actually it will be passed because there is no `UX` folder under it. I would like to remove `-Exclude`